charts/statping/README_CHANGELOG.md.gotmpl

```diff
@@ -0,0 +1,27 @@
+{{- define "custom.changelog.header" -}}
+## Changelog
+{{- end -}}
+
+{{- define "custom.changelog" -}}
+{{ template "custom.changelog.header" . }}
+
+All notable changes to this application Helm chart will be documented in this file but does not include changes from our common library. To read those click [here](https://github.com/k8s-at-home/charts/tree/master/charts/common/README.md#Changelog).
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+### [2.0.0]
+
+#### Added
+
+- N/A
+
+#### Changed
+
+- Redone based on common chart
```

Comment:
```suggestion
- Migrated to common library chart
```

Reply:
Might also be worth while noting that service.port has moved to service.port.port (as thats one of the bigger breaking changes)

Reply:
Added - thanks for the feedback